ci/test-checks.sh

```diff
@@ -15,6 +44,7 @@ cargoNightly="$(readlink -f "./cargo") nightly"
 
 scripts/increment-cargo-version.sh check
 
+if [ "$local" = false ] ; then
```

Comment:
what do you think if we check `CI` environment here? it looks like

```diff
- if [ "$local" = false ] ; then
+ if [ -n "$CI" ] ; then
```
and then we can remove the logic used to parse other arguments.

(btw, using `git diff` to capture uncommitted Cargo.lock changes seems to be a little bit severe)

Reply:
yeah. checking `$CI` is better.  updated.